components/authz-service/server/v2/project.go

```diff
@@ -440,6 +452,34 @@ func fromAPIProjectRuleConditionTypes(t api.ProjectRuleConditionTypes) (storage.
 	return 0, fmt.Errorf("invalid condition type %s", t.String())
 }
 
+var storageToAPIConditionOperators = map[storage.ConditionOperator]api.ProjectRuleConditionOperators{
+	storage.MemberOf: api.ProjectRuleConditionOperators_MEMBER_OF,
+	storage.Equals:   api.ProjectRuleConditionOperators_EQUALS,
+}
+
+var apiToStorageConditionOperators = map[api.ProjectRuleConditionOperators]storage.ConditionOperator{}
+var onceReverseConditionOperatorsMapping sync.Once
+
+func fromStorageConditionOperator(t storage.ConditionOperator) (api.ProjectRuleConditionOperators, error) {
+	if s, ok := storageToAPIConditionOperators[t]; ok {
+		return s, nil
+	}
+	return 0, fmt.Errorf("invalid condition operator %s", t.String())
+}
+
+func fromAPIProjectRuleConditionOperators(t api.ProjectRuleConditionOperators) (storage.ConditionOperator, error) {
+	onceReverseConditionOperatorsMapping.Do(func() {
+		for k, v := range storageToAPIConditionOperators {
+			apiToStorageConditionOperators[v] = k
+		}
+	})
+
+	if s, ok := apiToStorageConditionOperators[t]; ok {
+		return s, nil
+	}
+	return 0, fmt.Errorf("invalid condition operator %s", t.String())
```

Comment:
s/%s/%q/
(Also just noticed that the two lines for "invalid condition type" should have the same change.)